ios/app/JitsiMeetBroadcast Extension/SampleHandler.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  SampleHandler.swift
+//  Broadcast Extension
+//
+//  Created by Alex-Dan Bumbu on 22/03/2021.
+//  Copyright © 2021 Atlassian Inc. All rights reserved.
+//
+
+import ReplayKit
+
+private enum Constants {
+    static let appGroupIdentifier = "group.org.jitsi.meet.appgroup"
```

Comment:
Maybe put a comment mentioning this needs to be adapted for each app?

Reply:
done